### ISSUE TITLE
CodeView library Affect performance

### ISSUE THREAD

**Issue by shark5** [daquexian/FlexibleRichTextView#5]

**Comment by daquexian** [comment]
Just to clarify, do you notice the poor performance when you have a large part of code?

**Comment by shark5** [comment]
I used it in list item, and the phone is low-end device. It was GC all the time.

**Comment by hfbi360** [comment]
yes I encounter the poor performance in the low-end device too , but remove "CodeProcessor.init(this);" code ，it run good.
